continuous_integration/environment-3.10.yaml

```diff
@@ -0,0 +1,70 @@
+# This job includes coverage
+name: test-environment
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  # required dependencies
+  - python=3.10
+  - setuptools<60.0
```

Comment:
xref https://github.com/dask/distributed/pull/5923
can we get rid of this?

Reply:
Done